examples/multi-node-training/.deepspeed_env

```diff
@@ -0,0 +1,4 @@
+HCCL_OVER_TCP=0
```

Comment:
I don't think we need this. did you try without the above ?

Reply:
Actually I just copied what I found in the model reference example: https://github.com/HabanaAI/Model-References/tree/1.7.1/PyTorch/nlp/pretraining/deepspeed-bert#bert-15b-with-zero1-and-bf16-mixed-precision-on-32-hpus
I will try without :+1: